napari/_qt/qt_main_window.py

```diff
@@ -557,11 +560,29 @@ def closeEvent(self, event):
 
         Regardless of whether cmd Q, cmd W, or the close button is used...
         """
+        try:
+            status = self._plugin_manager_dialog.query_status()
+            is_plugin_manager_busy = status['status'] == PluginStatus.BUSY
+            plugin_manager_description = status['description']
+        except AttributeError:
```

Comment:
This is only expected while collecting `self._plugin_manager_dialog`, right? i.e. it shouldn't be silenced/ignored if `PluginStatus` ends up removing `BUSY`. Can we adjust it so the exception catching only covers the intended parts?

Reply:
@jaimergp if the user never opened the plugin dialog, then there is no status as self._plugin_manager_dialog is None, so that is why I left everything inside.

Does that make sense?

Reply:
Yes, but in that case the try clause only needs to wrap the first line, not number 2 and 3.